packages/worker/tsconfig.build.json

```diff
@@ -2,7 +2,10 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "./build",
-    "baseUrl": "./src"
+    "baseUrl": "./src",
+    "paths": {
+      "@gradejs-public/shared": ["../../shared/src"]
```

Comment:
The same here

Reply:
Good point, fixed in much more proper way.